### DISCUSSION TITLE
How can i play all the songs in library?

### DISCUSSION THREAD

**Discussion by Jantjuhhh** [music-assistant/support#2010] [Q&A] [answered]
First of all good work boys and girls, a stupid question maybe i've got a lot of music on my Homeassistant server and want to play all of my music in the library how to do that? 

greetz 
Jan

**Comment by marcelveldt** [comment]
This is not yet possible. You will have to create a playlist to do so.
We will offer some default playlists such as "all music" within a few weeks, before we launch stable.

**Reply by Jantjuhhh** [reply]
Ok then i am going to make a playlist, thnx for replying ;)

**Reply by OzGav** [reply, marked as answer]
A built in playlist now exists for this

**Reply by donburch888** [reply]
When you say "A built in playlist now exists for this" do you mean "now" as in the current version 2.0.0b143, or the next release ?

Also will Music Assistant play ALL tracks in the playlist one after another - or will it still play only ONE track chosen from the playlist ?

**Reply by OzGav** [reply]
Yes now. There are a couple of built in playlists from a few betas ago.

Not sure what you are referring to with your second question. A playlist by definition is a list where all the songs are played,

**Reply by Jantjuhhh** [reply]
Works like a charm ;) was waiting for it, phew and with the database update this is getting better and better.

greetings from a music addict
